src/components/Banner/MucBanner.vue

```diff
@@ -60,6 +66,18 @@ const typeAriaLabel = computed(() => {
       return "Information";
   }
 });
+
+const typeIcon = computed(() => {
+  switch (props.type) {
+    case "success":
+      return "#icon-check";
+    case "info":
+    case "warning":
+    case "emergency":
+    default:
+      return "#icon-information";
```

Comment:
```suggestion
    case "success":
      return "#icon-check";
    case "emergency":
      return "#icon-warning";
    case "info":
    case "warning":
    default:
      return "#icon-information";
```
Emergency has its own icon `!` instead of `i`

Reply:
actually, warning also has the exclamation mark - I didn't even notice this until now!

Reply:
true!

Reply:
Fixed with [03e307d](https://github.com/it-at-m/muc-patternlab-vue/pull/134/commits/03e307d32c802683a48e8b2b14c4189f8b10aedc)